lib/redux/actions/async.js

```diff
@@ -538,13 +582,37 @@ export function retrieveTargetsFromStorage() {
   };
 }
 
+export function createCustodialAccount(profile) {
+  return (dispatch, getState) => {
+    const { api } = services;
+    dispatch(syncActions.createCustodialAccountRequest());
+    api.user.createCustodialAccount(profile, (err, account) => {
+      if (err) {
+        dispatch(syncActions.createCustodialAccountFailure(err));
+      }
+      else {
+        dispatch(syncActions.createCustodialAccountSuccess(account));
```

Comment:
It is because we refresh everything, but I think it actually makes more sense for that success action to carry the payload of the newly created account (even if we're not currently listening for/using it anywhere yet), so I'll add the payload to that action.